projects/storage/toolbox/storage_download_to_image/vars/main/resources.yml

```diff
@@ -0,0 +1,4 @@
+---
+entrypoint: '{{ role_path }}/files/entrypoint.sh'
+pod_template: 'templates/pod.yml.j2'
+pvc_template: 'templates/pvc.yml.j2'
```

Comment:
from the [github action](https://github.com/openshift-psap/topsail/actions/runs/11943760517/job/33293409251?pr=594):
```
23:48:40 ERROR [validate_role_files.py:101] pvc_template: templates/pvc.yml.j2 --> not found
```
I think you don't want this line